hack/ccm/Makefile

```diff
@@ -0,0 +1,39 @@
+#TODO: Update the registry
+REGISTRY=quay.io/kabhat
+IMG=powervs-cloud-controller-manager
+
+# VPC_CONTROLLER_COMMIT can be fetched from here https://github.com/openshift/cloud-provider-vpc-controller/commits/master
+VPC_CONTROLLER_COMMIT?=9b99b4e
+# POWERVS_CLOUD_CONTROLLER_COMMIT can be fetched from here https://github.com/openshift/cloud-provider-powervs/commits/main
+POWERVS_CLOUD_CONTROLLER_COMMIT?=a6bfa07
+TAG?=$(VPC_CONTROLLER_COMMIT)_$(POWERVS_CLOUD_CONTROLLER_COMMIT)
+
+build-image-and-push-linux-amd64: init-buildx
+	{                                                                   \
+	set -e ;                                                            \
+	docker buildx build \
+		--build-arg TARGETPLATFORM=linux/amd64 --build-arg ARCH=amd64 \
+		--build-arg VPC_CONTROLLER_COMMIT=$(VPC_CONTROLLER_COMMIT) --build-arg POWERVS_CLOUD_CONTROLLER_COMMIT=$(POWERVS_CLOUD_CONTROLLER_COMMIT)\
+		-t $(REGISTRY)/$(IMG):$(TAG)_linux_amd64 . --push --target centos-base; \
+	}
+
+build-image-and-push-linux-ppc64le: init-buildx
+	{                                                                    \
+	set -e ;                                                             \
+	docker buildx build \
+		--build-arg TARGETPLATFORM=linux/ppc64le --build-arg ARCH=ppc64le\
+		--build-arg VPC_CONTROLLER_COMMIT=$(VPC_CONTROLLER_COMMIT) --build-arg POWERVS_CLOUD_CONTROLLER_COMMIT=$(POWERVS_CLOUD_CONTROLLER_COMMIT)\
+		-t $(REGISTRY)/$(IMG):$(TAG)_linux_ppc64le . --push --target centos-base; \
+	}
+
+init-buildx:
+	# Ensure we use a builder that can leverage it (the default on linux will not)
+	docker buildx rm multiarch-multiplatform-builder
+	docker buildx create --use --name=multiarch-multiplatform-builder
+	docker run --rm --privileged multiarch/qemu-user-static --reset --credential yes --persistent yes
+	# Required for "docker buildx build --push".
+	#TODO: Add respective registry login info
```

Comment:
you need `gcloud auth` command as well if used in the CI to build and push via buildx